tests/android/launch-server.spec.ts

```diff
@@ -20,26 +20,34 @@ import { androidTest as test, expect } from './androidTest';
 test('android.launchServer should connect to a device', async ({ playwright }) => {
   const browserServer = await playwright._android.launchServer();
   const device = await playwright._android.connect(browserServer.wsEndpoint());
+  const closeEvents = [];
+  browserServer.on('close', () => closeEvents.push('close'));
   const output = await device.shell('echo 123');
   expect(output.toString()).toBe('123\n');
   await device.close();
+  expect(closeEvents).toEqual(['close']);
```

Comment:
Why closing the connected device closes the server? Shouldn't I be able to connect to it again?